components/typography/metadata/typography.yml

```diff
@@ -42,22 +42,22 @@ examples:
 
       [View the Code typography component](typography-code.html)
     markup: |
-      <code class="spectrum-Code spectrum-Code--S">alert("Hello world");</code>
+      <code class="spectrum-Code spectrum-Code--sizeS">alert("Hello world");</code>
 sections:
   - name: Applying margins
     description: |
       By default, Typography components do not include outer margins. If you would like to add margins, simply add the `.spectrum-Typography` class to your container, and every typography component inside of your container will have the correct margins.
     markup: |
       <div class="spectrum-Typography">
-        <h1 class="spectrum-Heading spectrum-Heading--XL">Aliquet Mauris Eu</h1>
-        <p class="spectrum-Body spectrum-Body--XL">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eleifend est mollis ligula lobortis, tempus ultricies sapien lacinia. Nulla ut turpis velit. Sed finibus dapibus diam et sollicitudin. Phasellus in ipsum nec ante elementum congue eget in leo. Morbi eleifend justo non rutrum venenatis. Fusce cursus et lectus eu facilisis. Ut laoreet felis in magna dignissim feugiat.</p>
+        <h1 class="spectrum-Heading spectrum-Heading--sizeXL">Aliquet Mauris Eu</h1>
+        <p class="spectrum-Body spectrum-Body--sizeXL">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eleifend est mollis ligula lobortis, tempus ultricies sapien lacinia. Nulla ut turpis velit. Sed finibus dapibus diam et sollicitudin. Phasellus in ipsum nec ante elementum congue eget in leo. Morbi eleifend justo non rutrum venenatis. Fusce cursus et lectus eu facilisis. Ut laoreet felis in magna dignissim feugiat.</p>
       </div>
 
   - name: Migrating from deprecated Typography
```

Comment:
Can we also update this section to show the new-new typography classes, not the old-new? i.e.

```
| `~.spectrum-Heading1--display~`  | `.spectrum-Heading.spectrum-Heading--sizeXXXL`                        ```